app/redux/store.jsx

```diff
@@ -0,0 +1,25 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux'
+import createLogger from 'redux-logger'
+import thunkMiddleware from 'redux-thunk'
+
+import { whoami } from './reducers/auth'
+
+const rootReducer = combineReducers({
+  auth: require('./reducers/auth').default,
```

Comment:
I think it is easier to read when this key is 'currentUser' instead of 'auth' but its no biggie either way